solidity/test/utils/HyperlaneTestUtils.t.sol

```diff
@@ -28,10 +28,14 @@ abstract contract HyperlaneTestUtils is Test {
         uint64 timestamp,
         uint32 emitterChainId,
         bytes32 emitterAddress,
+        bytes32 merkleTreeHookAddress,
+        bytes32 root,
+        uint32 checkpointIndex,
+        bytes32 messageId,
         bytes memory payload,
         uint8 numSigners
     ) public view returns (bytes memory updateData) {
-        uint8 version = 1;
+        uint8 version = 3;
```

Comment:
same for the version

Reply:
Done, will apply further changes in the refacto branch ✅